docker/proxy/nginx.conf

```diff
@@ -0,0 +1,35 @@
+user nobody nogroup;
+worker_processes auto;
+
+events {
+  worker_connections 512;
+}
+
+http {
+
+  server {
+    listen 8080;
+    server_name "";
+
+    location / {
+      proxy_pass					http://docker2.pcic.uvic.ca:8000;
```

Comment:
Can you audit the config files and ensure that the docker host hostname (e.g. docker2.pcic) is templated out and not hard-coded? You can either use your existing j2 templating scheme or see the section "Using environment variables in nginx configuration" in the ["nginx docs"](https://hub.docker.com/r/_/nginx/).